relationships/synthesis/EXT-SERVICE-to-INFRA-ELASTICACHEREDISCLUSTER.yml

```diff
@@ -1,5 +1,29 @@
 relationships:
   - name: extServiceCallsInfraElastiCacheRedisCluster
+    version: "1"
+    origins:
+      - OpenTelemetry
+    conditions:
+      - attribute: eventType
+        anyOf: [ "Metric" ]
+      - attribute: db.system
+        anyOf: [ "redis" ]
+    relationship:
+      expires: P75M
+      relationshipType: CALLS
+      source:
+        extractGuid:
+          attribute: entity.guid
+      target:
+        lookupGuid:
+          candidateCategory: REDIS
+          fields:
+            - field: endpoint
+              attribute: net.peer.name
+            - field: port
+              attribute: net.peer.port
+
+  - name: extServiceCallsInfraElastiCacheRedisNode
```

Comment:
Aren't these the exact same rules? Is there isn't any difference, we only need one.